### PR TITLE
PARQUET-2103: Fix crypto exception in print toPrettyJSON

### PR DESCRIPTION
**Pull request by ggershinsky**

### CODE REVIEW

File: parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/FileMetaData.java [hunk 1]
```diff
@@ -71,7 +79,7 @@ public MessageType getSchema() {
 
   @Override
   public String toString() {
-    return "FileMetaData{schema: "+schema+ ", metadata: " + keyValueMetaData + "}";
+    return "FileMetaData{schema: "+schema+ ", metadata: " + keyValueMetaData + ", encryption: " + encryptionType + "}";
```
Comment (wgtmac) [inline parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/FileMetaData.java:82]:
```suggestion
    return "FileMetaData{schema: " + schema + ", metadata: " + keyValueMetaData + ", encryption: " + encryptionType + "}";
```

File: parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/FileMetaData.java [hunk 2]
```diff
@@ -50,16 +51,23 @@ public final class FileMetaData implements Serializable {
...
   public FileMetaData(MessageType schema, Map<String, String> keyValueMetaData, String createdBy) {
-    this(schema, keyValueMetaData, createdBy, null);
+    this(schema, keyValueMetaData, createdBy, EncryptionType.UNENCRYPTED, null);
   }
-  
-  public FileMetaData(MessageType schema, Map<String, String> keyValueMetaData, String createdBy, InternalFileDecryptor fileDecryptor) {
+
+  public FileMetaData(MessageType schema, Map<String, String> keyValueMetaData, String createdBy,
+                      InternalFileDecryptor fileDecryptor) {
+    this(schema, keyValueMetaData, createdBy, null, fileDecryptor);
```
Comment (wgtmac) [inline parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/FileMetaData.java:59]:
```suggestion
    this(schema, keyValueMetaData, createdBy, EncryptionType.UNENCRYPTED, fileDecryptor);
```

**Comment by shangxinli** [comment]
LGTM once the comments are addressed

File: parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/FileMetaData.java (hunk 2 above)
Comment (ggershinsky) [reply parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/FileMetaData.java:59]: The encryption type is unknown here, but this method is not called anymore, so I'll deprecate it.

**Comment by ggershinsky** [comment]
Thanks @shangxinli . @wgtmac , are you ok with the current state of the PR?

**Comment by wgtmac** [comment]
> Thanks @shangxinli . @wgtmac , are you ok with the current state of the PR?

LGTM. Feel free to merge. Thanks!

**Pull request merged**